packages/xod-func-tools/test/monads.spec.js

```diff
@@ -125,6 +126,26 @@ describe('moands', () => {
     );
   });
 
+  describe('maybeToPromise()', () => {
+    it('returns resolved promise contained Just value', () =>
+      maybeToPromise(
+        () => new Error('It should not been thrown'),
+        a => a,
+        Maybe.Just(52)
+      ).then(val => assert.equal(val, 52))
+    );
+    it('returns rejected promise', () =>
+      maybeToPromise(
+        () => new Error('It is Nothing!'),
+        a => a,
```

Comment:
please add a test where `justFn` is not an identity function

Reply:
Fixed! Thanks for the catch.